app.py

```diff
@@ -0,0 +1,634 @@
+                    dbc.Row(
+                        [
+                            dbc.Col(
+                                html.Img(
+                                    src="assets/dwave_logo.png", height="20px"
+                                ),
+                                style={"margin": "5px", "outline": False},
+                            ),
+                        ],
+                        align="center",
+                        className="g-0",
+                    ),
+                    href="https://dwavesys.com",
+                    style={
+                        "textDecoration": "none",
+                    },
+                )
+            ],
+            color="#074c91",
+        )
+    ]
+)
+
+
+title_card = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                dbc.Row(
+                    [
+                        html.H1(
+                            children="Employee Scheduling Demo",
+                            style={
+                                "textAlign": "center",
+                                "font-family": ff,
+                                "className": "border-0 bg-transparent",
+                                "color": col,
+                            },
+                        )
+                    ]
+                )
+            ]
+        )
+    ],
+    className="border-0 bg-transparent",
+)
+
+input_card = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.P(
+                    "Number of Employees:  ",
+                    style={"font-family": ff, "color": col},
+                ),
+                dbc.Input(
+                    id="input_employees",
+                    type="number",
+                    placeholder="#",
+                    min=4,
+                    max=200,
+                    step=1,
+                    value=12,
+                    style={"marginBottom": "5px", "outline": False},
+                ),
+                html.P(
+                    "Optional random seed: ",
+                    style={"font-family": ff, "color": col},
+                ),
+                dbc.Input(
+                    id="seed",
+                    type="number",
+                    placeholder="(Optional) Random Seed",
+                    min=1,
+                    style={"marginBottom": "5px", "outline": False},
+                ),
+                html.P(
+                    "Autofill scenario: ",
+                    style={"font-family": ff, "color": col},
+                ),
+                dcc.Dropdown(
+                    ["Small", "Medium", "Large"],
+                    placeholder="Select a scenario",
+                    id="demo-dropdown",
+                ),
+            ]
+        )
+    ],
+    className="border-0 bg-transparent",
+)
+
+settings_card = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                dbc.Button(
+                    "Additional Options",
```

Comment:
As I say in another comment, I think it would look nicer to use a tab to separate basic and additional configuration options. If you decide to stick with the collapsable button, I think it should at least use a different color to the "Solve CQM" button, which should stand out.  Currently the two buttons are identical:

![image](https://github.com/dwave-examples/employee-scheduling/assets/34041130/363624ac-b50d-4992-9f75-4f1d777e162b)